.github/workflows/release.yml

```diff
@@ -59,4 +59,10 @@ jobs:
           git commit -m "Create version v${{ inputs.version }}"
           git tag v${{ inputs.version }}
           git push origin release
-          git push origin v${{ inputs.version }}
\ No newline at end of file
+          git push origin v${{ inputs.version }}
+  call-image-release:
+    uses: ./.github/workflows/publish-image-bpn-discovery.yml
```

Comment:
file name is not correct. 
Please adjust the file to point to image-discovery-finder.yml

Reply:
Incorporated the review comments 
Adjusted the file name